saved-sites/saved-sites-impl/src/main/java/com/duckduckgo/savedsites/impl/sync/SavedSitesSyncDataProvider.kt

```diff
@@ -96,79 +100,53 @@ class SavedSitesSyncDataProvider @Inject constructor(
             return emptyList()
         }
 
-        val updates = mutableListOf<SyncBookmarkEntry>()
+        val updates = mutableListOf<SyncSavedSitesRequestEntry>()
         // favorites (we don't add individual items, they are added as we go through bookmark folders)
         if (hasFavorites) {
```

Comment:
Not really. For favourites we don’t add the bookmark itself, only the folders. This is because those are not recursively traversed, favourites don’t have folders.